cyberark-secrets-provider-for-kubernetes/templates/_helpers.tpl

```diff
@@ -0,0 +1,18 @@
+{{/*
+Return the most recent RBAC API available
+*/}}
+{{- define "conjur.rbac-api" -}}
+{{- if .Capabilities.APIVersions.Has "rbac.authorization.k8s.io/v1" }}
+{{- printf "rbac.authorization.k8s.io/v1" -}}
+{{- else if .Capabilities.APIVersions.Has "v1" }}
+{{- printf "v1" -}}
+{{- end }}
+{{- end }}
+
+{{- define "conjur.deploy-api" -}}
+{{- if .Capabilities.APIVersions.Has "batch/v1" }}
+{{- printf "batch/v1" -}}
+{{- else if .Capabilities.APIVersions.Has "apps/v1" }}
```

Comment:
Did you check this?
It looks weird to use `apps/v1` for jobs

Reply:
batch.v1 is for jobs